broker/lib/routes/admin.js

```diff
@@ -3,76 +3,77 @@
 const express = require('express');
 const config = require('../config');
 const middleware = require('../middleware');
+const common_middleware = require('../../../common/middleware');
```

Comment:
Can we not use class inheritance for middleware like Controllers? So that we do not have to require common middleware and modules' own middleware separately and use separate variables.